### PR TITLE
feat: new design for extension page from `extensions.json`

### PR DESCRIPTION
**Pull request by dni**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -8,3 +8,4 @@ logs
 reports
 jmeter.log
 upgrades
+metadata.json
```
Comment (motorina0) [inline .gitignore:11]: who produces this file?

Comment (dni) [reply .gitignore:11]: build.sh

Comment (motorina0) [reply .gitignore:11]: why doesn't it remove it when done?

**Review by motorina0 — CHANGES_REQUESTED** [review]
NACK
I think we should not introduce solidjs.

File: .gitignore (hunk 1 above)
Comment (dni) [reply .gitignore:11]: its cached so it does not hat every extension json every time

**Review by talvasconcelos — APPROVED** [review]

**Review by arcbtc — APPROVED** [review]

**Pull request merged**